### ISSUE TITLE
[wxwidgets] wxwidgets[core]:x64-windows -> 3.1.5#6 broken after latest changes

### ISSUE THREAD

**Issue by jpo234** [microsoft/vcpkg#22521]
**Describe the bug**
Install is broken after upgrade to wxwidgets[core]:x64-windows -> 3.1.5#6

**Environment**
- OS: Win10
- Compiler: Visual Studio 2019

**To Reproduce**

```
C:\src\vcpkg>vcpkg install wxwidgets:x64-windows
Computing installation plan...
The following packages will be built and installed:
    wxwidgets[core]:x64-windows -> 3.1.5#6
Detecting compiler hash for triplet x64-windows...
-- Using HTTP(S)_PROXY in environment variables.
Restored 0 packages from C:\Users\jpo\AppData\Local\vcpkg\archives in 223.7 us. Use --debug to see more details.
Starting package 1/1: wxwidgets:x64-windows
Building package wxwidgets[core]:x64-windows...
-- Using cached wxWidgets-wxWidgets-9c0a8be1dc32063d91ed1901fd5fcd54f4f955a1.tar.gz.
-- Extracting source C:/src/vcpkg/downloads/wxWidgets-wxWidgets-9c0a8be1dc32063d91ed1901fd5fcd54f4f955a1.tar.gz
-- Applying patch disable-platform-lib-dir.patch
-- Applying patch fix-build.patch
-- Using source at C:/src/vcpkg/buildtrees/wxwidgets/src/54f4f955a1-0d7c66a59e.clean
-- Found external ninja('1.10.2').
-- Configuring x64-windows
-- Building x64-windows-dbg
-- Building x64-windows-rel
-- Installing: C:/src/vcpkg/packages/wxwidgets_x64-windows/share/wxwidgets/copyright
-- Performing post-build validation
-- Performing post-build validation done
Stored binary cache: C:\Users\jpo\AppData\Local\vcpkg\archives\9a\9a8c3a297ad07d9e7d03b00cbbe2fa0541d0d9faf23340281c868c26168237a4.zip
Installing package wxwidgets[core]:x64-windows...
Elapsed time for package wxwidgets:x64-windows: 7.241 min

Total elapsed time: 7.304 min

The package wxwidgets provides CMake integration:

    find_package(wxWidgets REQUIRED)
    target_include_directories(main PRIVATE ${wxWidgets_INCLUDE_DIRS})
    target_link_libraries(main PRIVATE ${wxWidgets_LIBRARIES})
```

**Expected behavior**
wxWidgets can be used

**Failure logs** 
`2>C:\src\vcpkg\installed\x64-windows\include\wx\setup.h(2,10): fatal error C1083: Cannot open include file: '../../debug/lib//setup.h': No such file or directory
`
**Additional context**
Used to work with previous version

**Comment by jpo234** [comment]
Did a clean reinstall of vcpkg, still messed up:

```
C:\src>git clone https://github.com/Microsoft/vcpkg.git
Cloning into 'vcpkg'...
remote: Enumerating objects: 138375, done.
remote: Counting objects: 100% (94/94), done.
remote: Compressing objects: 100% (21/21), done.
remote: Total 138375 (delta 81), reused 73 (delta 73), pack-reused 138281 eceiving objects: 100% (138375/138375), 48.79 MiB | 1.25 MiB/s
Receiving objects: 100% (138375/138375), 48.88 MiB | 613.00 KiB/s, done.
Resolving deltas: 100% (86079/86079), done.
Updating files: 100% (8489/8489), done.

C:\src>.\vcpkg\bootstrap-vcpkg.bat
Downloading https://github.com/microsoft/vcpkg-tool/releases/download/2021-12-09/vcpkg.exe -> C:\src\vcpkg\vcpkg.exe (using proxy: http://proxyhost)... done.
Validating signature... done.

Telemetry
---------
vcpkg collects usage data in order to help us improve your experience.
The data collected by Microsoft is anonymous.
You can opt-out of telemetry by re-running the bootstrap-vcpkg script with -disableMetrics,
passing --disable-metrics to vcpkg on the command line,
or by setting the VCPKG_DISABLE_METRICS environment variable.

Read more about vcpkg telemetry at docs/about/privacy.md

C:\src>cd vcpkg

C:\src\vcpkg>vcpkg integrate install
Applied user-wide integration for this vcpkg root.

All MSBuild C++ projects can now #include any installed libraries.
Linking will be handled automatically.
Installing new libraries will make them instantly available.

CMake projects should use: "-DCMAKE_TOOLCHAIN_FILE=C:/src/vcpkg/scripts/buildsystems/vcpkg.cmake"

C:\src\vcpkg>vcpkg install wxwidgets
Computing installation plan...
A suitable version of cmake was not found (required v3.21.1). Downloading portable cmake v3.21.1...
Downloading cmake...
  https://github.com/Kitware/CMake/releases/download/v3.21.1/cmake-3.21.1-windows-i386.zip -> C:\src\vcpkg\downloads\cmake-3.21.1-windows-i386.zip
Extracting cmake...
A suitable version of 7zip was not found (required v19.0.0). Downloading portable 7zip v19.0.0...
Downloading 7zip...
  https://www.7-zip.org/a/7z1900-x64.msi -> C:\src\vcpkg\downloads\7z1900-x64.msi
Extracting 7zip...
The following packages will be built and installed:
  * expat[core]:x64-windows -> 2.4.1
  * libjpeg-turbo[core]:x64-windows -> 2.1.2
  * liblzma[core]:x64-windows -> 5.2.5#4
  * libpng[core]:x64-windows -> 1.6.37#16
  * tiff[core,jpeg,lzma,zip]:x64-windows -> 4.3.0#3
  * vcpkg-cmake[core]:x64-windows -> 2021-12-20
  * vcpkg-cmake-config[core]:x64-windows -> 2021-12-28
    wxwidgets[core]:x64-windows -> 3.1.5#6
  * zlib[core]:x64-windows -> 1.2.11#13
Additional packages (*) will be modified to complete this operation.
Detecting compiler hash for triplet x64-windows...
-- Using HTTP(S)_PROXY in environment variables.
A suitable version of powershell-core was not found (required v7.2.0). Downloading portable powershell-core v7.2.0...
Downloading powershell-core...
  https://github.com/PowerShell/PowerShell/releases/download/v7.2.0/PowerShell-7.2.0-win-x86.zip -> C:\src\vcpkg\downloads\PowerShell-7.2.0-win-x86.zip
Extracting powershell-core...
Restored 0 packages from C:\Users\jpo\AppData\Local\vcpkg\archives in 439.6 us. Use --debug to see more details.
Starting package 1/9: expat:x64-windows
Building package expat[core]:x64-windows...
-- Downloading https://github.com/libexpat/libexpat/archive/a28238bdeebc087071777001245df1876a11f5ee.tar.gz -> libexpat-libexpat-a28238bdeebc087071777001245df1876a11f5ee.tar.gz...
-- Extracting source C:/src/vcpkg/downloads/libexpat-libexpat-a28238bdeebc087071777001245df1876a11f5ee.tar.gz
-- Applying patch pkgconfig.patch
-- Using source at C:/src/vcpkg/buildtrees/expat/src/876a11f5ee-539179dcdb.clean
-- Found external ninja('1.10.2').
-- Configuring x64-windows
-- Building x64-windows-dbg
-- Building x64-windows-rel
-- Fixing pkgconfig file: C:/src/vcpkg/packages/expat_x64-windows/lib/pkgconfig/expat.pc
-- Downloading https://repo.msys2.org/mingw/i686/mingw-w64-i686-pkg-config-0.29.2-2-any.pkg.tar.zst;https://www2.futureware.at/~nickoe/msys2-mirror/mingw/i686/mingw-w64-i686-pkg-config-0.29.2-2-any.pkg.tar.zst;https://mirror.yandex.ru/mirrors/msys2/mingw/i686/mingw-w64-i686-pkg-config-0.29.2-2-any.pkg.tar.zst;https://mirrors.tuna.tsinghua.edu.cn/msys2/mingw/i686/mingw-w64-i686-pkg-config-0.29.2-2-any.pkg.tar.zst;https://mirrors.ustc.edu.cn/msys2/mingw/i686/mingw-w64-i686-pkg-config-0.29.2-2-any.pkg.tar.zst;https://mirror.bit.edu.cn/msys2/mingw/i686/mingw-w64-i686-pkg-config-0.29.2-2-any.pkg.tar.zst;https://mirror.selfnet.de/msys2/mingw/i686/mingw-w64-i686-pkg-config-0.29.2-2-any.pkg.tar.zst;https://mirrors.sjtug.sjtu.edu.cn/msys2/mingw/i686/mingw-w64-i686-pkg-config-0.29.2-2-any.pkg.tar.zst -> msys-mingw-w64-i686-pkg-config-0.29.2-2-any.pkg.tar.zst...
-- Downloading https://repo.msys2.org/mingw/i686/mingw-w64-i686-libwinpthread-git-8.0.0.5906.c9a21571-1-any.pkg.tar.zst;https://www2.futureware.at/~nickoe/msys2-mirror/mingw/i686/mingw-w64-i686-libwinpthread-git-8.0.0.5906.c9a21571-1-any.pkg.tar.zst;https://mirror.yandex.ru/mirrors/msys2/mingw/i686/mingw-w64-i686-libwinpthread-git-8.0.0.5906.c9a21571-1-any.pkg.tar.zst;https://mirrors.tuna.tsinghua.edu.cn/msys2/mingw/i686/mingw-w64-i686-libwinpthread-git-8.0.0.5906.c9a21571-1-any.pkg.tar.zst;https://mirrors.ustc.edu.cn/msys2/mingw/i686/mingw-w64-i686-libwinpthread-git-8.0.0.5906.c9a21571-1-any.pkg.tar.zst;https://mirror.bit.edu.cn/msys2/mingw/i686/mingw-w64-i686-libwinpthread-git-8.0.0.5906.c9a21571-1-any.pkg.tar.zst;https://mirror.selfnet.de/msys2/mingw/i686/mingw-w64-i686-libwinpthread-git-8.0.0.5906.c9a21571-1-any.pkg.tar.zst;https://mirrors.sjtug.sjtu.edu.cn/msys2/mingw/i686/mingw-w64-i686-libwinpthread-git-8.0.0.5906.c9a21571-1-any.pkg.tar.zst -> msys-mingw-w64-i686-libwinpthread-git-8.0.0.5906.c9a21571-1-any.pkg.tar.zst...
-- Using msys root at C:/src/vcpkg/downloads/tools/msys2/aa5af7b2aa7e90e8
-- Fixing pkgconfig file: C:/src/vcpkg/packages/expat_x64-windows/debug/lib/pkgconfig/expat.pc
-- Installing: C:/src/vcpkg/packages/expat_x64-windows/share/expat/copyright
-- Performing post-build validation
-- Performing post-build validation done
Stored binary cache: C:\Users\jpo\AppData\Local\vcpkg\archives\2d\2d695da96590c10d403ed6fe059d1eee4d4defa9e5c337e2e7fefd13e1715642.zip
Installing package expat[core]:x64-windows...
Elapsed time for package expat:x64-windows: 45.75 s
Starting package 2/9: vcpkg-cmake:x64-windows
Building package vcpkg-cmake[core]:x64-windows...
-- Installing: C:/src/vcpkg/packages/vcpkg-cmake_x64-windows/share/vcpkg-cmake/vcpkg_cmake_configure.cmake
-- Installing: C:/src/vcpkg/packages/vcpkg-cmake_x64-windows/share/vcpkg-cmake/vcpkg_cmake_build.cmake
-- Installing: C:/src/vcpkg/packages/vcpkg-cmake_x64-windows/share/vcpkg-cmake/vcpkg_cmake_install.cmake
-- Installing: C:/src/vcpkg/packages/vcpkg-cmake_x64-windows/share/vcpkg-cmake/vcpkg_cmake_get_vars.cmake
-- Installing: C:/src/vcpkg/packages/vcpkg-cmake_x64-windows/share/vcpkg-cmake/cmake_get_vars
-- Installing: C:/src/vcpkg/packages/vcpkg-cmake_x64-windows/share/vcpkg-cmake/cmake_get_vars/CMakeLists.txt
-- Installing: C:/src/vcpkg/packages/vcpkg-cmake_x64-windows/share/vcpkg-cmake/vcpkg-port-config.cmake
-- Installing: C:/src/vcpkg/packages/vcpkg-cmake_x64-windows/share/vcpkg-cmake/copyright
-- Performing post-build validation
-- Performing post-build validation done
Stored binary cache: C:\Users\jpo\AppData\Local\vcpkg\archives\06\06d46e1ef5265c3908a87f365655f6c4e7e5bac4755aaf013629e5e112ae8e54.zip
Installing package vcpkg-cmake[core]:x64-windows...
Elapsed time for package vcpkg-cmake:x64-windows: 210.3 ms
Starting package 3/9: vcpkg-cmake-config:x64-windows
Building package vcpkg-cmake-config[core]:x64-windows...
-- Installing: C:/src/vcpkg/packages/vcpkg-cmake-config_x64-windows/share/vcpkg-cmake-config/vcpkg_cmake_config_fixup.cmake
-- Installing: C:/src/vcpkg/packages/vcpkg-cmake-config_x64-windows/share/vcpkg-cmake-config/vcpkg-port-config.cmake
-- Installing: C:/src/vcpkg/packages/vcpkg-cmake-config_x64-windows/share/vcpkg-cmake-config/copyright
-- Performing post-build validation
-- Performing post-build validation done
Stored binary cache: C:\Users\jpo\AppData\Local\vcpkg\archives\9b\9b05bf747dfefb6e01fc04681f875920d040a2a2694f74c93c11533874a55c5a.zip
Installing package vcpkg-cmake-config[core]:x64-windows...
Elapsed time for package vcpkg-cmake-config:x64-windows: 157.7 ms
Starting package 4/9: libjpeg-turbo:x64-windows
Building package libjpeg-turbo[core]:x64-windows...
-- Downloading https://github.com/libjpeg-turbo/libjpeg-turbo/archive/ecf021bc0d6f435daacff7c35ccaeef0145df1b9.tar.gz -> libjpeg-turbo-libjpeg-turbo-ecf021bc0d6f435daacff7c35ccaeef0145df1b9.tar.gz...
-- Extracting source C:/src/vcpkg/downloads/libjpeg-turbo-libjpeg-turbo-ecf021bc0d6f435daacff7c35ccaeef0145df1b9.tar.gz
-- Applying patch add-options-for-exes-docs-headers.patch
-- Applying patch workaround_cmake_system_processor.patch
-- Using source at C:/src/vcpkg/buildtrees/libjpeg-turbo/src/f0145df1b9-3892709aaa.clean
-- Downloading https://www.nasm.us/pub/nasm/releasebuilds/2.15.05/win32/nasm-2.15.05-win32.zip;https://fossies.org/windows/misc/nasm-2.15.05-win32.zip -> nasm-2.15.05-win32.zip...
-- Found external ninja('1.10.2').
-- Configuring x64-windows
-- Building x64-windows-dbg
-- Building x64-windows-rel
-- Fixing pkgconfig file: C:/src/vcpkg/packages/libjpeg-turbo_x64-windows/lib/pkgconfig/libjpeg.pc
-- Fixing pkgconfig file: C:/src/vcpkg/packages/libjpeg-turbo_x64-windows/lib/pkgconfig/libturbojpeg.pc
-- Using cached msys-mingw-w64-i686-pkg-config-0.29.2-2-any.pkg.tar.zst.
-- Using cached msys-mingw-w64-i686-libwinpthread-git-8.0.0.5906.c9a21571-1-any.pkg.tar.zst.
-- Using msys root at C:/src/vcpkg/downloads/tools/msys2/aa5af7b2aa7e90e8
-- Fixing pkgconfig file: C:/src/vcpkg/packages/libjpeg-turbo_x64-windows/debug/lib/pkgconfig/libjpeg.pc
-- Fixing pkgconfig file: C:/src/vcpkg/packages/libjpeg-turbo_x64-windows/debug/lib/pkgconfig/libturbojpeg.pc
-- Installing: C:/src/vcpkg/packages/libjpeg-turbo_x64-windows/share/libjpeg-turbo/copyright
-- Performing post-build validation
-- Performing post-build validation done
Stored binary cache: C:\Users\jpo\AppData\Local\vcpkg\archives\c3\c3682dce43248b70af692509d0c6a9029068888ba9074c7b686f774a6cd35dde.zip
Installing package libjpeg-turbo[core]:x64-windows...
Elapsed time for package libjpeg-turbo:x64-windows: 45.47 s
Starting package 5/9: liblzma:x64-windows
Building package liblzma[core]:x64-windows...
-- Downloading https://github.com/xz-mirror/xz/archive/v5.2.5.tar.gz -> xz-mirror-xz-v5.2.5.tar.gz...
-- Extracting source C:/src/vcpkg/downloads/xz-mirror-xz-v5.2.5.tar.gz
-- Applying patch enable-uwp-builds.patch
-- Applying patch fix_config_include.patch
-- Applying patch win_output_name.patch
-- Applying patch add_support_ios.patch
-- Using source at C:/src/vcpkg/buildtrees/liblzma/src/v5.2.5-d69c6caafe.clean
-- Found external ninja('1.10.2').
-- Configuring x64-windows
-- Building x64-windows-dbg
-- Building x64-windows-rel
-- Fixing pkgconfig file: C:/src/vcpkg/packages/liblzma_x64-windows/lib/pkgconfig/liblzma.pc
-- Using cached msys-mingw-w64-i686-pkg-config-0.29.2-2-any.pkg.tar.zst.
-- Using cached msys-mingw-w64-i686-libwinpthread-git-8.0.0.5906.c9a21571-1-any.pkg.tar.zst.
-- Using msys root at C:/src/vcpkg/downloads/tools/msys2/aa5af7b2aa7e90e8
-- Fixing pkgconfig file: C:/src/vcpkg/packages/liblzma_x64-windows/debug/lib/pkgconfig/liblzma.pc
-- Installing: C:/src/vcpkg/packages/liblzma_x64-windows/share/liblzma/copyright
-- Performing post-build validation
-- Performing post-build validation done
Stored binary cache: C:\Users\jpo\AppData\Local\vcpkg\archives\cb\cb024625a0e098dc677cdd7cee639d513e3991108c74e19b11ae1dce0d5bf863.zip
Installing package liblzma[core]:x64-windows...
Elapsed time for package liblzma:x64-windows: 24.08 s
Starting package 6/9: zlib:x64-windows
Building package zlib[core]:x64-windows...
-- Downloading https://www.zlib.net/zlib-1.2.11.tar.gz;https://downloads.sourceforge.net/project/libpng/zlib/1.2.11/zlib-1.2.11.tar.gz -> zlib1211.tar.gz...
-- Extracting source C:/src/vcpkg/downloads/zlib1211.tar.gz
-- Applying patch cmake_dont_build_more_than_needed.patch
-- Applying patch 0001-Prevent-invalid-inclusions-when-HAVE_-is-set-to-0.patch
-- Applying patch add_debug_postfix_on_mingw.patch
-- Applying patch 0002-android-build-mingw.patch
-- Using source at C:/src/vcpkg/buildtrees/zlib/src/1.2.11-e9a093319e.clean
-- Found external ninja('1.10.2').
-- Configuring x64-windows
-- Building x64-windows-dbg
-- Building x64-windows-rel
-- Installing: C:/src/vcpkg/packages/zlib_x64-windows/share/zlib/vcpkg-cmake-wrapper.cmake
-- Fixing pkgconfig file: C:/src/vcpkg/packages/zlib_x64-windows/lib/pkgconfig/zlib.pc
-- Using cached msys-mingw-w64-i686-pkg-config-0.29.2-2-any.pkg.tar.zst.
-- Using cached msys-mingw-w64-i686-libwinpthread-git-8.0.0.5906.c9a21571-1-any.pkg.tar.zst.
-- Using msys root at C:/src/vcpkg/downloads/tools/msys2/aa5af7b2aa7e90e8
-- Fixing pkgconfig file: C:/src/vcpkg/packages/zlib_x64-windows/debug/lib/pkgconfig/zlib.pc
-- Installing: C:/src/vcpkg/packages/zlib_x64-windows/share/zlib/copyright
-- Performing post-build validation
-- Performing post-build validation done
Stored binary cache: C:\Users\jpo\AppData\Local\vcpkg\archives\91\91fbed57f0b8845e0e12f909102c7bff0e70c7b92697680ae92b8c9f2227d855.zip
Installing package zlib[core]:x64-windows...
Elapsed time for package zlib:x64-windows: 10.84 s
Starting package 7/9: libpng:x64-windows
Building package libpng[core]:x64-windows...
-- Downloading https://github.com/glennrp/libpng/archive/v1.6.37.tar.gz -> glennrp-libpng-v1.6.37.tar.gz...
-- Extracting source C:/src/vcpkg/downloads/glennrp-libpng-v1.6.37.tar.gz
-- Applying patch use_abort.patch
-- Applying patch cmake.patch
-- Applying patch pkgconfig.patch
-- Applying patch pkgconfig.2.patch
-- Applying patch fix-export-targets.patch
-- Applying patch macos-arch-fix.patch
-- Using source at C:/src/vcpkg/buildtrees/libpng/src/v1.6.37-05658d6f1a.clean
-- Found external ninja('1.10.2').
-- Configuring x64-windows
CMake Warning at installed/x64-windows/share/vcpkg-cmake/vcpkg_cmake_configure.cmake:487 (message):
  The following variables are not used in CMakeLists.txt:

      PNG_ARM_NEON

  Please recheck them and remove the unnecessary options from the
  `vcpkg_cmake_configure` call.

  If these options should still be passed for whatever reason, please use the
  `MAYBE_UNUSED_VARIABLES` argument.
Call Stack (most recent call first):
  ports/libpng/portfile.cmake:71 (vcpkg_cmake_configure)
  scripts/ports.cmake:142 (include)


-- Building x64-windows-dbg
-- Building x64-windows-rel
-- Fixing pkgconfig file: C:/src/vcpkg/packages/libpng_x64-windows/lib/pkgconfig/libpng.pc
-- Fixing pkgconfig file: C:/src/vcpkg/packages/libpng_x64-windows/lib/pkgconfig/libpng16.pc
-- Using cached msys-mingw-w64-i686-pkg-config-0.29.2-2-any.pkg.tar.zst.
-- Using cached msys-mingw-w64-i686-libwinpthread-git-8.0.0.5906.c9a21571-1-any.pkg.tar.zst.
-- Using msys root at C:/src/vcpkg/downloads/tools/msys2/aa5af7b2aa7e90e8
-- Fixing pkgconfig file: C:/src/vcpkg/packages/libpng_x64-windows/debug/lib/pkgconfig/libpng.pc
-- Fixing pkgconfig file: C:/src/vcpkg/packages/libpng_x64-windows/debug/lib/pkgconfig/libpng16.pc
-- Installing: C:/src/vcpkg/packages/libpng_x64-windows/share/libpng/copyright
-- Performing post-build validation
-- Performing post-build validation done
Stored binary cache: C:\Users\jpo\AppData\Local\vcpkg\archives\ae\ae515e05da1fbc94f7518c07d1e27e697181560c34300875790ea6f84c408725.zip
Installing package libpng[core]:x64-windows...
Elapsed time for package libpng:x64-windows: 14.91 s
Starting package 8/9: tiff:x64-windows
Building package tiff[core,jpeg,lzma,zip]:x64-windows...
-- Downloading https://gitlab.com/libtiff/libtiff/-/archive/v4.3.0/libtiff-v4.3.0.tar.gz -> libtiff-libtiff-v4.3.0.tar.gz...
-- Extracting source C:/src/vcpkg/downloads/libtiff-libtiff-v4.3.0.tar.gz
-- Applying patch cmakelists.patch
-- Applying patch fix-pkgconfig.patch
-- Applying patch FindCMath.patch
-- Using source at C:/src/vcpkg/buildtrees/tiff/src/v4.3.0-f4a22a2fc5.clean
-- Found external ninja('1.10.2').
-- Configuring x64-windows
-- Building x64-windows-dbg
-- Building x64-windows-rel
-- Fixing pkgconfig file: C:/src/vcpkg/packages/tiff_x64-windows/lib/pkgconfig/libtiff-4.pc
-- Using cached msys-mingw-w64-i686-pkg-config-0.29.2-2-any.pkg.tar.zst.
-- Using cached msys-mingw-w64-i686-libwinpthread-git-8.0.0.5906.c9a21571-1-any.pkg.tar.zst.
-- Using msys root at C:/src/vcpkg/downloads/tools/msys2/aa5af7b2aa7e90e8
-- Fixing pkgconfig file: C:/src/vcpkg/packages/tiff_x64-windows/debug/lib/pkgconfig/libtiff-4.pc
-- Installing: C:/src/vcpkg/packages/tiff_x64-windows/share/tiff/copyright
-- Performing post-build validation
-- Performing post-build validation done
Stored binary cache: C:\Users\jpo\AppData\Local\vcpkg\archives\65\657d99e1df76aada5f0fea4b8792dc971bde6879aaa552992f420de72c870ef4.zip
Installing package tiff[core,jpeg,lzma,zip]:x64-windows...
Elapsed time for package tiff:x64-windows: 30 s
Starting package 9/9: wxwidgets:x64-windows
Building package wxwidgets[core]:x64-windows...
-- Downloading https://github.com/wxWidgets/wxWidgets/archive/9c0a8be1dc32063d91ed1901fd5fcd54f4f955a1.tar.gz -> wxWidgets-wxWidgets-9c0a8be1dc32063d91ed1901fd5fcd54f4f955a1.tar.gz...
-- Extracting source C:/src/vcpkg/downloads/wxWidgets-wxWidgets-9c0a8be1dc32063d91ed1901fd5fcd54f4f955a1.tar.gz
-- Applying patch disable-platform-lib-dir.patch
-- Applying patch fix-build.patch
-- Using source at C:/src/vcpkg/buildtrees/wxwidgets/src/54f4f955a1-0d7c66a59e.clean
-- Found external ninja('1.10.2').
-- Configuring x64-windows
-- Building x64-windows-dbg
-- Building x64-windows-rel
-- Installing: C:/src/vcpkg/packages/wxwidgets_x64-windows/share/wxwidgets/copyright
-- Performing post-build validation
-- Performing post-build validation done
Stored binary cache: C:\Users\jpo\AppData\Local\vcpkg\archives\4c\4ce63d30f1b092b54f7959798299a25f20a32d83cce29c94b1f3f62642e47d79.zip
Installing package wxwidgets[core]:x64-windows...
Elapsed time for package wxwidgets:x64-windows: 7.582 min

Total elapsed time: 12.92 min

The package wxwidgets provides CMake integration:

    find_package(wxWidgets REQUIRED)
    target_include_directories(main PRIVATE ${wxWidgets_INCLUDE_DIRS})
    target_link_libraries(main PRIVATE ${wxWidgets_LIBRARIES})


```

**Comment by jpo234** [comment]
> Fix is ready: #22509

Still broken in a new way. Generated setup.h now looks like this:

```
#ifdef _DEBUG
#include "../../debug/lib/C:/src/vcpkg/packages/wxwidgets_x64-windows/debug/lib/mswud/wx/setup.h"
#else
#include "../../lib/C:/src/vcpkg/packages/wxwidgets_x64-windows/lib/mswu/wx/setup.h"
#endif

```
